coding_style/README.adoc

```diff
@@ -292,10 +292,9 @@ Alternatively, most configuration files have their own modules, such as https://
 Using these make code cleaner to read and ensure idempotence.
 ====
 
-* Limit use of the `copy` module to copying remote files and to uploading binary blobs.
-  For all other file pushes, use the `template` module.
-  Even if there is nothing in the file that is being templated at the current moment, having the file handled by the `template` module now makes adding that functionality much simpler than if the file is initially handled by the `copy` and then needs to be moved before it can be edited.
-* When using the `template` module, append `.j2` to the template file name. 
+* Limit use of the `copy` module to copying remote files, static files, and to uploading binary blobs.
+For most file pushes, use the `template` module. Even if there currently is nothing in the file that is being templated, if there is the possibility in the future that it might be added, having the file handled by the `template` module now makes adding that functionality much simpler than if the file is initially handled by the copy and then needs to be moved before it can be edited.
```

Comment:
```suggestion
  For most file pushes, use the `template` module.
  Even if there currently is nothing in the file that is being templated, if there is the possibility in the future that it might be added, having the file handled by the `template` module now makes adding that functionality much simpler than if the file is initially handled by the `copy` module and then needs to be moved before it can be edited.
```

Reply:
let me know if the current version fixed the issues you saw with the original